t/mojo/subprocess.t

```diff
@@ -62,6 +62,18 @@ Mojo::IOLoop->start;
 ok !$fail, 'no error';
 is_deeply $result, ['♥', [{two => 2}], 3], 'right structure';
 
+# Promises
```

Comment:
You didn't test the `Mojo::IOLoop->subprocess->run_p(...)` variant.

Reply:
Didn't see how it would be any different, but I've added a test now.